packages/dashboard/src/app/views/exploreTemplates/header/index.js

```diff
@@ -31,20 +32,37 @@ import { useFeature } from 'flagged';
 import {
   DASHBOARD_VIEWS,
   TEMPLATES_GALLERY_SORT_MENU_ITEMS,
+  TEXT_INPUT_DEBOUNCE,
 } from '../../../../constants';
 import {
   FilterPropTypes,
   SortPropTypes,
   ViewPropTypes,
+  SearchPropTypes,
 } from '../../../../utils/useTemplateView';
 import { useDashboardResultsLabel } from '../../../../utils';
 import { PageHeading, BodyViewOptions } from '../../shared';
 
-function Header({ filter, isLoading, totalTemplates, sort, view }) {
+function Header({
+  filter,
+  isLoading,
+  totalTemplates,
+  sort,
+  view,
+  search,
+  searchOptions = [],
+}) {
   const enableInProgressTemplateActions = useFeature(
```

Comment:
Feels weird using this experiment for the search input since this is tied to the old sort feature (that could probably just get gutted). Could you add a new isolated experiment for `enableExploreTemplatesSearch`?